tests/types/t8781.nim

```diff
@@ -0,0 +1,25 @@
+discard """
```

Comment:
This is a `ccgbugs` test, not a `types` one.

Reply:
Thanks, I didn't know that.